client/src/component/Button/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button } from '@material-ui/core';
+
+const ButtonComponent = ({ variant, color, children, ...rest }) => (
+  <>
```

Comment:
What is the purpose of using fragment since it's a single component?

Reply:
Done